junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/extension/TempDirectory.java

```diff
@@ -190,8 +190,41 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) {
 				private FileVisitResult deleteAndContinue(Path path) {
 					try {
 						Files.delete(path);
+						return CONTINUE;
 					}
 					catch (IOException ex) {
+						return deleteUndeletableAndContinue(path, ex);
+					}
```

Comment:
Can we move `return CONTINUE;` to the end of this method and make `deleteUndeletableAndContinue` return `void`?

Reply:
I've made it return Throwable, so it is always recorded in a single place